src/java/org/apache/cassandra/schema/TableId.java

```diff
@@ -170,7 +164,8 @@ public UUID asUUID()
     @Override
     public int hashCode()
     {
-        return Hashing.hash(msb ^ lsb);
+        long hilo = msb ^ lsb;
+        return ((int)(hilo >> 32)) ^ (int) hilo;
```

Comment:
reverted to the previous logic.  Reading the code `Hashing.hash` would alter the hash code, so figured safest to just keep it matching `UUID`